.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,8 @@
 <!--
-    OpenShift: Don't forget to run `./scripts/rh-manifest.sh` from the
```

Comment:
is this intentional?

Reply:
I asked ART if we can get rid of that manifest.

Reply:
Good catch there! Once we can cofirm that the cachito integration is done from their side we can get rid of it

Reply:
nope, will fix it